cfme/tests/storage/test_volume_types.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+from cfme import test_requirements
+from cfme.cloud.provider.ec2 import EC2Provider
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.blockers import BZ
+from cfme.utils.providers import list_providers_by_class
+
+
+pytestmark = [
+    pytest.mark.tier(3),
+    test_requirements.storage,
+    pytest.mark.usefixtures("setup_provider_modscope"),
+    pytest.mark.provider([EC2Provider, OpenStackProvider], scope="module"),
+]
+
+
+def assert_volume_type_should_be_present(block_manager, should_be_present):
+    view = navigate_to(block_manager, 'Details')
+    block_storage_manager_relationship_fields = view.entities.relationships.fields
+    assert (
+        ('Cloud Volume Types' in block_storage_manager_relationship_fields) == should_be_present)
+
+
+@pytest.mark.meta(automates=[1650082], blockers=[
+    BZ(1650082, unblock=lambda provider: not provider.one_of(OpenStackProvider))])
```

Comment:
@mmojzis I think the tests are running against OSP because this BZ is not blocking, you may need `forced_streams`.

Reply:
I had "5.11" in `forced_streams`, but didn't help...But will try again...

Reply:
@john-dupuy Added `forced_streams` and OpenStack is still collected.